inc/commonitilobject.class.php

```diff
@@ -6710,15 +6710,14 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
          $total_actiontime = 0;
 
          $criteria = [
-            'SELECT'   => 'actiontime',
-            'DISTINCT' => true,
+            'SELECT'   => ['SUM' => 'actiontime AS actiontime'],
```

Comment:
I would use a different name for AS to prevent confusion, but this is not a blocker